src/cascade/input_data/db/csmr.py

```diff
@@ -29,13 +30,36 @@ def _csmr_in_t3(execution_context):
     return exists == 1
 
 
+def _gbd_process_version_id_from_cod_version(cod_version):
+    query = """
+    SELECT gbd.gbd_process_version_metadata.gbd_process_version_id from gbd.gbd_process_version_metadata
+        JOIN  gbd.gbd_process_version ON gbd.gbd_process_version_metadata.gbd_process_version_id =
+                                         gbd.gbd_process_version.gbd_process_version_id
+        WHERE gbd_process_id = 3                     -- codcorrect process
+              and metadata_type_id = 1               -- codcorrect version
+              and val = %(cod_version)s
+              and gbd_process_version_status_id = 1  -- marked best
+    """
+
+    with cursor(database="gbd") as c:
+        c.execute(query, args={"cod_version": cod_version})
+        result = c.fetchone()
+
+    if result is None:
+        raise ValueError(f"No best gbd_process_version_id for cod version {cod_version}")
+
+    return result[0]
+
+
 def _get_csmr_data(execution_context):
 
     cause_id = execution_context.parameters.add_csmr_cause
     parent_loc = execution_context.parameters.location_id
 
     keep_cols = ["year_id", "location_id", "sex_id", "age_group_id", "val", "lower", "upper"]
 
+    process_version_id = _gbd_process_version_id_from_cod_version(execution_context.parameters.cod_version)
```

Comment:
Maybe we should add some comments here to say a little something about what process_version_id is?